lib/restrictions.js

```diff
@@ -73,6 +73,25 @@ export async function insert({restriction} = {}) {
   return record;
 }
 
+/**
+ * Inserts multiple restrictions into database.
+ *
+ * @param {object} options - The options to use.
+ * @param {Array} options.restrictions - An array of restrictions.
+ * @returns {Promise} - Settles once the operation completes.
+ */
+export async function bulkInsert({restrictions} = []) {
```

Comment:
I think this is a minor mistake here.

You are using object destructuring on an array when I believe you want:

```js
bulkInsert({restrictions = []} = {}) {
```
that will mean if no options are specified a blank object is provided,
then restrictions will default to an empty array.

Reply:
We should add `assert.array` here instead of setting a default, IMO.